modules/custom/activity_send/modules/activity_send_email/src/Controller/ActivitySendEmailController.php

```diff
@@ -16,6 +16,12 @@ class ActivitySendEmailController extends ControllerBase {
    *
    * @return \Symfony\Component\HttpFoundation\RedirectResponse
    *   Return Redirect to the user account.
+   *
+   * @deprecated in open_social:13.0.0 and is removed from open_social:14.0.0.
```

Comment:
We can deprecate it, but propose to use `\Drupal\user\Controller\UserController::userEditPage` instead of another custom one.

Reply:
reverted